spec/units/peep_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'peep'
+require 'database_helpers'
+
+describe Peep do
+  describe '.all' do
+    it 'returns all peeps' do
+      connection = PG.connect(dbname: 'chitter_test')
+
+      peep = Peep.create(message: 'Peep 1', date: '2021-06-25')
```

Comment:
I like the way that you have stored the first test case in a variable to check the assertions later on in a more convenient way.